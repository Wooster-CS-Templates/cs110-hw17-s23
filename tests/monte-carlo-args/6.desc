Run command: ./monte-carlo abc def ghi

