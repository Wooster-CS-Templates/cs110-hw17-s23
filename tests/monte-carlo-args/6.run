./monte-carlo abc def ghi