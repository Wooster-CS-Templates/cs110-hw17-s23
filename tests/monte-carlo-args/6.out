The provided values must be positive integers
